.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: File a bug report
+title: 
+labels: 
+assignees: 
+
+---
+## What version of Rust are you on?
+...
+
+## Problem
+...
+
+## Stacktrace
```

Comment:
Rust calls them backtraces and they only manifest if something panics. Not if it it's a compile-time problem or if if it's a `Result` that returns an `Err` though there may be context on that error.

Maybe `## Backtrace (if applicable)`?